app/src/main/java/com/brewfind/dto/Brewery.kt

```diff
@@ -2,7 +2,11 @@ package com.brewfind.dto
 
 import com.google.gson.annotations.SerializedName
 
-data class Brewery(@SerializedName("Name") var name: String, @SerializedName("brewery_type") var size: String, @SerializedName("street") var street: String) {
+data class Brewery(@SerializedName("name") var name: String,
```

Comment:
Change the capitalization of the value name, since it's lowercase in the API it references and to match the other value names.

Reply:
The @SerializedName typically matches an external data source, like a JSON stream.  So, only change this if that matches the data in a JSON data stream that is being consumed here.